README.md

```diff
@@ -9,3 +9,47 @@ In the case of runc containers, the VF is directly usable by the container.
 
 In the case of clear containers, if clear containers detect a VF in the namespace, the runtime will unbind the VF from the host, bind it to VFIO and assign it to the clear container using PCI device pass thro.
 The clear container runtime which support SRIOV can be found at https://github.com/01org/cc-oci-runtime/tree/networking/sriov
+
+# How to use this plugin
+
+
+0. Build this plugin.
+
+        go build
+
+1. Ensure that your plugin is discoverable https://docs.docker.com/engine/extend/plugin_api/#/plugin-discovery
+
+        sudo cp sriov.json /etc/docker/plugins
+
+
+2. Start the plugin
+
+        sudo ./sriov &
+
+3. Create a mapping file to relate the network interface name of the SRIOV NIC Physical Function (PF) and the name of physical network the NIC is part of. 
+   For example, the network interface name corresponding to PF is eth0 and the NIC is part of the physical network b2b
```

Comment:
@sainath14 I think we should just pass this as an option to the docker network creation logic. This will make it cleaner. We can revisit this.